modules/logwriter/tests/logwriter.tftest.hcl

```diff
@@ -0,0 +1,15 @@
+run "setup" {
+  module {
+    source = "../testing/run"
+  }
+}
+
+run "install_logwriter" {
+  variables {
+    name                    = run.setup.short
+    bucket_arn              = run.setup.bucket_arn
+    discovery_rate          = "10 minutes"
+    filter_name             = "${run.setup.id}-filter"
+    log_group_name_patterns = ["*"]
```

Comment:
Restrict this to `${run.setup.id}`. That way you ensure you only subscribe to log groups created as part of this run, which will inevitably get deleted during test teardown.

Reply:
i initially had this and also `run.setup.short` and both fail the validation. is there a way to see the state for the run somehow?